rdbms/src/test/java/io/spine/server/storage/jdbc/delivery/JdbcInboxStorageTest.java

```diff
@@ -202,19 +201,19 @@ private void markDelivered(ImmutableList<InboxMessage> messages) {
     private static ImmutableList<InboxMessage>
     readAllAndCompare(InboxStorage storage, ShardIndex idx, ImmutableList<InboxMessage> expected) {
         Page<InboxMessage> page = storage.readAll(idx, Integer.MAX_VALUE);
-        assertEquals(expected.size(), page.size());
+        assertThat(page.size()).isEqualTo(expected.size());
 
         ImmutableList<InboxMessage> contents = page.contents();
-        assertEquals(ImmutableSet.copyOf(expected), ImmutableSet.copyOf(contents));
+        assertThat(ImmutableSet.copyOf(contents)).isEqualTo(ImmutableSet.copyOf(expected));
         return contents;
     }
 
     private static void checkEmpty(InboxStorage storage, ShardIndex index) {
         Page<InboxMessage> emptyPage = storage.readAll(index, Integer.MAX_VALUE);
-        assertEquals(0, emptyPage.size());
-        assertTrue(emptyPage.contents()
-                            .isEmpty());
-        assertFalse(emptyPage.next()
-                             .isPresent());
+        assertThat(emptyPage.size()).isEqualTo(0);
+        assertThat(emptyPage.contents())
+                .isEmpty();
```

Comment:
Move this to one line. Same below.

Reply:
You won't believe. But this is how it is supposed to be :)